.travis.yml

```diff
@@ -9,6 +9,21 @@ php:
   - 7.1
   - hhvm
 
-before_script: composer install
+# This triggers builds to run on the new TravisCI infrastructure.
+# See: http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+sudo: false
+
+## Cache composer
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+matrix:
+  include:
+    - php: 5.3
+      env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+
+before_script:
+  - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-dist
```

Comment:
`--prefer-dist` is not a default, neither is `--prefer-source`. Composer automatically guesses what will be the fastest. For stable tags, that means dist, and for dev commits, that means it uses source, so that it can fetch changes on updates, instead of a full download.

In the context of CI, it makes sense, if no caching is used, to use dist all the time. Since we have only stable dependencies, there's little point in messing with the already good enough defaults.